docs/source/specifications/driver-override.rst

```diff
@@ -0,0 +1,46 @@
+.. _DriverOverride:
+
+Driver Override Specification
+==============================
+This document describes the specification for the global override feature.
+By default, every user is allowed write access to the devices by the master driver. The override feature will allow the user (for example, building administrator) to override this default behavior and enable the user to lock the write access on the devices for specified duration of time .
+
+Functional Capabilities
+-----------------------------
+
+1. User shall be able to specify the following when turning on the override behavior on the devices.
+
+    * Override pattern, for example,
+
+         If pattern is campus/building1/* - Override condition is turned on for all the devices under campus/building1
```

Comment:
We might also mention that pattern matching use bash style filename matching semantics so that it is clear that it's not regex.